.rubocop.yml

```diff
@@ -1,47 +1,87 @@
 inherit_from: .rubocop_todo.yml
+
+inherit_mode:
+  merge:
+    - Exclude
+
 AllCops:
-  TargetRubyVersion: 2.6
-  TargetRailsVersion: 5.2
-  Exclude:
-    - bin/*
-    - db/schema.rb
-    - db/migrate/*
-    - config/initializers/*
-    - 'vendor/**/*'
-    - 'lib/tasks/cucumber.rake'
-    - 'app/controllers/products_controller.rb'
-Rails:
-  Enabled: true
-Rails/FindEach: # Find_each does not work with mongoid
-  Enabled: false
-Rails/HasAndBelongsToMany:
-  Exclude:
-    - 'app/models/role.rb'
-Documentation:
-  Enabled: false
-Metrics/LineLength:
+  RubyInterpreters:
+    - ruby
+    - macruby
+    - rake
+    - jruby
+    - rbx
+  # Include common Ruby source files.
+  Exclude:
+      - bin/*
+      - db/schema.rb
+      - db/migrate/*
+      - config/initializers/*
+      - 'vendor/**/*'
+      - 'lib/tasks/cucumber.rake'
+  NewCops: enable
+  TargetRubyVersion: 2.7
+Layout/LineLength:
   Max: 150
   Exclude:
+    - 'app/controllers/products_controller.rb'
     - 'lib/cypress/data_criteria_attribute_builder.rb'
     - 'lib/cypress/demographics_randomizer.rb'
     - 'lib/validators/program_criteria_validator.rb'
     - 'test/**/*'
+Lint/MissingSuper:
+  Exclude:
+    - 'lib/cypress/highlighting/highlighting.rb'
+    - 'lib/validators/cat3_population_validator.rb'
+    - 'lib/validators/checklist_criteria_validator.rb'
+    - 'lib/validators/cms_schematron_validator.rb'
+    - 'lib/validators/core_clinical_data_element_validator.rb'
+    - 'lib/validators/ehr_certification_id_validator.rb'
+    - 'lib/validators/encounter_validator.rb'
+    - 'lib/validators/expected_results_validator.rb'
+    - 'lib/validators/measure_period_validator.rb'
+    - 'lib/validators/program_criteria_validator.rb'
+    - 'lib/validators/program_validator.rb'
+    - 'lib/validators/provider_type_validator.rb'
+    - 'lib/validators/qrda_cat1_validator.rb'
+    - 'lib/validators/qrda_cat3_validator.rb'
+    - 'lib/validators/qrda_upload_validator.rb'
+Lint/UnreachableLoop:
+  Exclude:
+    - 'test/unit/lib/record_filter_test.rb'    
```

Comment:
I think we should actually fix this one - either by turning the loop at line 225 into a .first or otherwise correcting it to the intended logic.